Tools/build/deepfreeze.py

```diff
@@ -513,7 +514,7 @@ def main() -> None:
     if args.file:
         if verbose:
             print(f"Reading targets from {args.file}")
-        with open(args.file, "rt", encoding="utf-8-sig") as fin:
+        with open(args.file, "rt", encoding="utf-8-sig") as fin:  # noqa: UP015
```

Comment:
Why noqa this?

Reply:
I... think I misread it with "truncate" and not text mode... and thought it was a false positive :')